examples/nek5000/make-nek-examples.sh

```diff
@@ -67,7 +67,7 @@ sed -i.bak -e "s|^#FC=.*|FC=\"$FC\"|" \
     -e "s|^#CC=.*|CC=\"$CC\"|" \
     -e "s|^#SOURCE_ROOT=.*|SOURCE_ROOT=\"$NEK5K_DIR\"|" \
     -e "s|^#FFLAGS=.*|FFLAGS+=\"-I${CEED_DIR}/include\"|" \
-    -e "s|^#USR_LFLAGS+=.*|USR_LFLAGS+=\"-L${CEED_DIR}/lib -lceed\"|" makenek
+    -e "s|^#USR_LFLAGS+=.*|USR_LFLAGS+=\"-L${CEED_DIR}/lib -Wl,-rpath,${CEED_DIR}/lib -lceed -fopenmp -lm\"|" makenek
```

Comment:
Did you intentionally re-add the `-fopenmp -lm` that @thilinarmtb removed as an undocumented side-effect of 262a692c69733cca68d74174699138f2bd553ee5? What should it be?

Reply:
No. I'm not sure why we didn't get a conflict -- Thilina removed them, while I was adding rpath (needed on my Linux box)

Reply:
They should be removed (I could build without them) and tzanio fixed already.

Reply:
Note that your need for RPATH is because `run-nek-example.sh` did not set `LD_LIBRARY_PATH` correctly (for an installed libceed). Now with RPATH, it shouldn't need to set mess with LD_LIBRARY_PATH at all. @thilinarmtb Can we remove that line?

    export LD_LIBRARY_PATH=`cd ../../lib; pwd`:${LD_LIBRARY_PATH}

Reply:
Yes, we can remove that. I was going to fix it in the next PR but I pushed that change anyway.